lib/xmlSchemaFaker.js

```diff
@@ -6,15 +6,18 @@ function convertSchemaToXML(name, schema, attribute, indentChar, inden) {
     cInden = _.times(inden, _.constant(indentChar)).join('');
   name = _.get(schema, 'xml.name', name || 'element');
   if (_.get(schema, 'xml.prefix')) {
-    tagPrefix = schema.xml.prefix + ':';
+    tagPrefix = schema.xml.prefix ? `${schema.xml.prefix}:` : '';
   }
-  if (['integer','string'].includes(schema.type)) {
+  if (['integer','string', 'boolean'].includes(schema.type)) {
```

Comment:
Should we take for for number as the possible types are https://json-schema.org/understanding-json-schema/reference/type.html
?